nsx-PowerOps.ps1

```diff
@@ -818,6 +829,425 @@ function parseSSHOutput ($fileToParse, $findElements, $direction) {
 #     [System.Console]::SetWindowPosition(0,[System.Console]::CursorTop)
 # }
 
+function runLB2Excel($sectionNumber){
+
+    $DocumentPath = "$documentlocation\LBToExcel-{0:yyyy}-{0:MM}-{0:dd}_{0:HH}-{0:mm}.xlsx" -f (get-date)
+
+    $nsxEdges = Get-NsxEdge | ? {$_.features.loadBalancer.enabled -eq "true"}
+    $allNSXComponentExcelDataEdge =@{}
+
+    #Some variables we will use to populate the summary page
+    $counterVIPs = 0
+    $counterVIPsOpen = 0
+    $counterVIPsClosed = 0
+    $counterPools = 0
+    $counterPoolsUp = 0
+    $counterPoolsDown = 0
+    $counterMembers = 0
+    $counterMembersUp = 0
+    $counterMembersDown = 0
+    $counterMembersUnknown = 0
+
+    $summaryEdgeStats = [ordered]@{}
+
+    # Create the new excel workbook that we will be working on
+    $nsxLBExcelWorkBook = createNewExcel
+
+    # Creating seperate worksheet for each edge lb
+    foreach ($edge in $nsxEdges) {
+
+        Write-Progress -Activity "Procesing $($edge.name)"
```

Comment:
if you want to ensure there is evidence of this being run in the event logs for a schedule task (non interactive) use out-event -entrytype information|warning|error - suggest you put at least one informational in here...